dask-gateway-dev.yaml

```diff
@@ -0,0 +1,36 @@
+# include the environment variable GO111MODULE=auto during installation for 
+# go 1.16 and up
+name: dask-gateway-dev
+channels:
+  - conda-forge
+dependencies:
+  - python=3.9
+  - pip  
+  - pytest
+  - go=1.17.8
+  - go-cgo
+  - compilers
+  - cryptography
+  - tornado
+  - traitlets
+  - sqlalchemy
+  - dask
+  - skein
+  - sphinx
+  - dask-sphinx-theme
+  # dask-gateway/continuous_integration/install.sh
+  - ipywidgets
+  - jupyterhub
+  - notebook
+  - trustme
+  # dask-gateway/continuous_integration/kubernetes/install.sh
+  - aiohttp
+  - black
+  - colorlog
+  - distributed
+  - flake8
+  - pytest-asyncio=0.12.0
```

Comment:
would also add kubectl and helm, which are necessary to do anything meaningful with k8s

Reply:
I will add those. Thanks!

Reply:
Added.